File: PHP PowerPoint Examples/SlidesApiDemo/deleteSlideBackground.php
<?php

use \Spire\Cloud\Powerpoint\Sdk\Api\SlidesApi;
use \Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new SlidesApi($configuration);

$name = "DeleteSlideBackground.pptx"; // string | Document name.
$password = null; // string | Document password.
$folder = "Slides"; // string | Document folder.
$storage = null; // string | Document storage.
$slideIndex = 1;
$apiInstance->deleteSlideBackground($name, $slideIndex, $password, $folder, $storage);

File: Python PowerPoint Examples/SlidesApiDemo/deleteSlideBackground.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.slides_api.SlidesApi(configuration)

fileName = "DeleteSlideBackground.pptx"
password = ""
storage = ""
slide_index = 1
inputFolder = "input"
result =api.delete_slide_background(fileName, slide_index=slide_index, password=password,storage=storage, folder=inputFolder)